redpen-core/src/main/java/cc/redpen/RedPen.java

```diff
@@ -138,7 +139,6 @@ public Document parse(DocumentParser parser, String content) throws RedPenExcept
     public Map<Document, List<ValidationError>> validate(List<Document> documents) {
         Map<Document, List<ValidationError>> docErrorsMap = new HashMap<>();
         documents.forEach(e -> docErrorsMap.put(e, new ArrayList<>()));
-        initializeValidators();
```

Comment:
Thank you for the fix of UI @gerryhocks! But this line is needed to fix https://github.com/redpen-cc/redpen/issues/621